config/passport.js

```diff
@@ -0,0 +1,10 @@
+var passport = require('passport')
+  , User = require('../models/user')
+  , LocalStrategy = require('passport-local').Strategy;
```

Comment:
That's probably the most maintainable way, but too ugly.

The only bad thing about this comma-first style is when you delete the last line, remember to add a new semicolon. It doesn't sacrifice much maintainability.